index.html

```diff
@@ -60,18 +60,268 @@ <h2 class="secondary-text">I'm a Software Developer</h2>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h3 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h3>
+                    <p class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </p>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        See Project
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h3 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h3>
+                    <p class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </p>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        See Project
+                    </div>
+                </div>
+            </div>
+        </div>
+        <section class="main-container">
+            <article class="about-container">
+                <article class="about">
+                        <h2 class="about-p-text" >About Me</h2>
+                        <p class="about-s-text">Hello I’m a software developer! I can 
+                            help you build a product , feature or 
+                            website Look through some of my work 
+                            and experience! If you like what you 
+                            see and have a project you need coded, 
+                            don’t hesitate to contact me.
+                        </p>
+                        <div class="btn-holder">
+                            <button class="btn">Get My Resume</button>
+                        </div>
+                        
+                        <img class="about-me-banner" src="/assets/images/About Me Background.svg" alt="Aboutmebanner">
```

Comment:
- [ ] Good job implementing the changes requested by the previous reviewer, however, you have somehow mistakenly removed the dot `.` in before all of your image tags. Please resolve those by putting `.` before the forward slashes. 

For example:
***Yours***
> src="/assets/images/About Me Background.svg" alt="Aboutmebanner">

***Possible Solution***

src="./assets/images/About Me Background.svg" alt="Aboutmebanner">

Please implement this for all other broken `<img>` tags